src/InfiniteOpt.jl

```diff
@@ -5,7 +5,6 @@ import Reexport
 Reexport.@reexport using JuMP
 
 # Import the necessary packages.
-import MathOptInterface
```

Comment:
This is probably because you're using `MathOptInterface` somewhere? It should be sufficient to just use the `MOI` symbol exported from JuMP.

Reply:
Ya that was my original thought, but I remembered that because I use `MOI.Utilities.CleverDict` I have method definitions for `MOI.Utilities.index_to_key` and `MOI.Utilities.key_to_index`. Thus, it would appear I need to maintain `MathOptInterface` as a dependency.

Reply:
I think you can still define methods using the `MOI` symbol. You can also pass functions to CleverDict instead of using the default fallbacks.

Reply:
Good catch, you are exactly right. I had an explicit `import MathOptInterface` in a submodule I didn't catch. Now it seems to work! Thank you for pointing me in the right direct.